.github/BOTMETA.yml

```diff
@@ -976,6 +976,8 @@ files:
     maintainers: elad661
   $modules/remote_management/manageiq/manageiq_group.py:
     maintainers: evertmulder
+  $modules/remote_management/manageiq/manageiq_provider.py:
+    maintainers: russoz $team_manageiq
```

Comment:
```suggestion
    maintainers: russoz $team_manageiq
  $modules/remote_management/manageiq/manageiq_provider_info.py:
    maintainers: russoz $team_manageiq
```

Reply:
I will stick with the _info module only ;-)